src/plugins/share/public/components/url_panel_content.test.tsx

```diff
@@ -77,7 +77,7 @@ describe('share url panel content', () => {
       component.find(EuiRadioGroup).prop('onChange')!(ExportUrlAsType.EXPORT_URL_AS_SAVED_OBJECT);
     });
     expect(component.find(EuiCopy).prop('textToCopy')).toEqual(
-      'http://localhost:5601/app/myapp#/?_g=()'
+      'http://localhost:5601/app/myapp#/?_g=%28%29'
```

Comment:
Huh, interesting - was this a subtle encoding bug previously?

Reply:
YES. Seems `url.format` won't convert these characters. HIs behavior is more like `encodeURIComponent` and the  result is not RFC3986-compliant.